src/crypto/tls/common.go

```diff
@@ -462,6 +463,12 @@ const (
 	ECDSAWithSHA1 SignatureScheme = 0x0203
 )
 
+type TLSFlag uint16
+
+const (
+	FlagSupportMTLS TLSFlag = 0x50
```

Comment:
Please document what this is. IIRC this is not yet a codepoint that's documented? If so, we should mark it as experimental in the API.
```suggestion
        /// ExperimentalFlagSuppportMTLS ...
	ExperimentalFlagSupportMTLS TLSFlag = 0x50
```